content/en/blog/_posts/2022-12-06-kubernetes-1.25-blog.md

```diff
@@ -0,0 +1,190 @@
+
+The Kubernetes project recommends using CRI version `v1`; in Kubernetes v1.25 the kubelet could also negotiate the use of CRI `v1alpha2` (which was deprecated along at the same time as adding support for the stable `v1` interface).
+
+Kubernetes v1.26 dropped support for CRI `v1alpha2`. That [removal](https://github.com/kubernetes/kubernetes/pull/110618) will result in the kubelet not registering the node if the container runtime doesn't support CRI `v1`. This means that containerd minor version 1.5 and older will not be supported in Kubernetes 1.26; if you use containerd, you will need to upgrade to containerd version 1.6.0 or later **before** you upgrade that node to Kubernetes v1.26. Other container runtimes that only support the `v1alpha2` are equally affected: if that affects you, you should contact the container runtime vendor for advice or check their website for additional instructions in how to move forward.
+
+### Signing Kubernetes release artifacts graduates to Stable [SIG Release]
+
+Introduced in [v1.24](https://kubernetes.io/blog/2022/05/03/kubernetes-1-24-release-announcement/), this feature constitutes a significant milestone in improving the security of the Kubernetes release process. All release artifacts are signed using [cosign](https://github.com/sigstore/cosign/), and both binary artifacts and images [can be verified](https://kubernetes.io/docs/tasks/administer-cluster/verify-signed-images/).
+
+### Improvements to Kubernetes metrics
+
+This release has several noteworthy improvements on metrics.
+
+#### Metrics framework extension graduates to Alpha
+
+The **metrics framework extension** graduates to Alpha and documentation can now be [auto-generated](https://kubernetes.io/docs/reference/instrumentation/metrics/) for every metric in the Kubernetes codebase, improving the ability to identify the metrics exposed by different Kubernetes components.
+
+#### Component Service Level Indicators graduates to Alpha
+
+Also improving on the ability to consume Kubernetes metrics, the **component Service Level Indicators (SLIs)** has graduated to Alpha: by enabling the `ComponentSLIs` feature flag there will be an [additional metrics endpoint](https://github.com/kubernetes/kubernetes/pull/112884) which allows the calculation of Service Level Objectives (SLOs) from raw healthcheck data converted into metric format.
+
+#### Feature metrics are now available
+
+**Feature metrics** are now available for each Kubernetes component, making it possible to [track whether each active feature gate is enabled](https://github.com/kubernetes/kubernetes/pull/112690) by checking the component's metric endpoint for `kubernetes_feature_enabled`.
+
+#### API server tracing
+
+_FIXME: API server tracing graduates to Beta [SIG Instrumentation] need clarification, https://github.com/kubernetes/enhancements/issues/647 indicates this is not tracked for 1.26_
```

Comment:
This isn't done.

Reply:
Indeed, will remove it - only kept it there because it was flagged as a major theme in the discussion and wanted to confirm.